azkaban-web-server/src/test/resources/image_management/create_image_rampup.json

```diff
@@ -0,0 +1,16 @@
+{
+  "planName": "Rampup plan 1",
+  "imageType": "spark_job",
+  "description": "Ramp up for spark job",
+  "activatePlan": true,
+  "imageRampups": [
+    {
+      "imageVersion": "1.6.2",
+      "rampupPercentage": "70"
+    },
+    {
+      "imageVersion": "1.6.1",
+      "rampupPercentage": "30"
+    }
+  ]
+}
```

Comment:
Nit: Please apply save action which will add new line here. Same for all the json files.

Reply:
Done.